src/change-hub.js

```diff
@@ -29,10 +33,20 @@ export async function changeHub(hubId, addToHistory = true, waypoint = null) {
     console.log("Change hub called with the current hub id. This is a noop.");
     return;
   }
+
   // Suppress on-screen join and leave messages until we receive a sync.
   APP.hideHubPresenceEvents = true;
   const scene = AFRAME.scenes[0];
 
+  // Generate leave events for everyone in the room.
```

Comment:
Why do we need to synthetically do this? Since we area actually leaving the room shouldn't we already get these for real?

Reply:
No -- the people we leave behind will receive a leave event for the client changing hub, but we don't receive leave events for everyone else. (This might be an artifact of how `migrateToHub` / `migrateToChannel` is implemented.)

Related, I've added this `TODO` about using `migrateToHub` in `changeHub`:

```
    // TODO Migrating to a new hub in one step makes state cleanup suspicious.
    //      Would prefer to disconnect, cleanup state, then connect to the new hub.
```